examples/evm-to-evm-fungible-transfer/README.md

```diff
@@ -37,7 +37,7 @@ yarn install
 To start the example you need to build the sdk first with:
 
 ```bash
-yarn sdk:build
+yarn build:all
```

Comment:
do we need to add yarn install as well?

Reply:
yes I believe this readme assumes you have global yarn installation

Reply:
ah but, if someones clones the repo and runs `yarn install` it will install this dependencies, nvm then